imports/client/components/AccountForm.tsx

```diff
@@ -272,18 +302,18 @@ class AccountForm extends React.Component {
           <h3>{title}</h3>
         </div>
         <div>
-          <form id="at-pwd-form" noValidate="" action="#" method="POST" onSubmit={this.submitForm}>
+          <form id="at-pwd-form" noValidate action="#" method="POST" onSubmit={this.submitForm}>
             <fieldset>
-              {this.state.submitState === 'failed' ? <Alert bsStyle="danger">{this.state.errorMessage}</Alert> : null}
-              {this.state.submitState === 'success' && this.state.successMessage ? <Alert bsStyle="success">{this.state.successMessage}</Alert> : null}
-              {format === 'login' || format === 'requestPwReset' ? emailInput : null}
-              {format === 'login' || format === 'enroll' || format === 'resetPwd' ? pwInput : null}
-              {format === 'login' ? pwResetOptionComponent : null}
-              {format === 'enroll' ? enrollmentFields : null}
+              {this.state.submitState === AccountFormSubmitState.FAILED ? <Alert bsStyle="danger">{this.state.errorMessage}</Alert> : null}
+              {this.state.submitState === AccountFormSubmitState.SUCCESS && this.state.successMessage ? <Alert bsStyle="success">{this.state.successMessage}</Alert> : null}
+              {format === AccountFormFormat.LOGIN || format === AccountFormFormat.REQUEST_PW_RESET ? emailInput : null}
+              {format === AccountFormFormat.LOGIN || format === AccountFormFormat.ENROLL || format === AccountFormFormat.RESET_PWD ? pwInput : null}
+              {format === AccountFormFormat.LOGIN ? pwResetOptionComponent : null}
+              {format === AccountFormFormat.ENROLL ? enrollmentFields : null}
```

Comment:
FWIW, I was a little surprised that it wasn't a type error to compare a string against an enum. I think it's allowed because the enum is assignable to the string (c.f. [the language spec](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#4.19.3))